tailwind.js

```diff
@@ -54,21 +53,22 @@ let colors = {
   'primary-lightest': '#D5FFF4',
   'primary-lighter': '#E6F5F0',
   'primary-light': '#69D5B1',
-  'primary': '#00bc89',
+  'primary-blur': '#D5FFF4',
```

Comment:
Please remove `primary-blur` and use `primary-lightest` as is the same value.

Reply:
Done!